src/axom/core/docs/sphinx/core_numerics.rst

```diff
@@ -0,0 +1,60 @@
+.. ## Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core numerics
+******************************************************
+
+The `axom::numerics` namespace was designed for convenient representation
+and use of a mathematical matrix, with accompanying manipulation and 
+solver routines.
+
+As an example, the following code shows vector operations.
+
+.. literalinclude:: ../../examples/core_numerics.cpp
+   :start-after: _vecops_start
+   :end-before: _vecops_end
+   :language: C++
+
+This example code shows how to construct a matrix.
+
+.. literalinclude:: ../../examples/core_numerics.cpp
+   :start-after: _matctor_start
+   :end-before: _matctor_end
+   :language: C++
+
+We can add and multiply matrices, vectors, and scalars, find the determinant,
+and extract upper and lower triangular matrices.
+
+.. literalinclude:: ../../examples/core_numerics.cpp
+   :start-after: _matops_start
+   :end-before: _matops_end
+   :language: C++
+
+We can use the power method or the Jacobi method to find the eigenvalues and
+vectors of a matrix.  Currently, the API of `eigen_solve` is not consistent
```

Comment:
Suggestion: Convert "Currently, ..." to a `note:` ?

Reply:
done